CHANGES.rst

```diff
@@ -18,6 +18,10 @@ dev (master)
 
 * Lazily load `uuid` to boost performance on imports (Pull #1270)
 
+* Allow providing a list of headers to strip from requests when redirecting
```

Comment:
Maybe worth adding another entry at the top of the CHANGES list that explicitly mentions the breaking change.

Reply:
I can add this, I also want to make mention to that this only occurs when the hostname in the redirect is different than the original requests hostname.